packages/layout/src/arkitemaCard/arkitemaCard.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+import { ArkitemaCard } from '.'
+import { BrowserRouter as Router } from 'react-router-dom'
+import KronborgStrandby from '@arkitema/brand'
```

Comment:
If the KronborgStrandBy image only are used in this story, then the image shouldn't be located in `@arkitema/brand` but be in the `.storybook/asset` folder.
See docs: https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook-configuration